legacy/src/api/accounts/businesses.md

```diff
@@ -51,13 +51,13 @@ A Business is associated with a single [Account][].
 <a name="onboarding-statuses">
 {% h4 Onboarding Statuses %}
 
-|    Status    |                                                                Description                                                                 |                                      Allowed Reasons                                      |
-| :----------- | :----------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------- |
+|    Status    |                                                                 Description                                                                 |                                      Allowed Reasons                                      |
+| :----------- | :------------------------------------------------------------------------------------------------------------------------------------------ | :---------------------------------------------------------------------------------------- |
 | applied      | Identifies businesses that have registered to be activated for the Centrapay service.                                                       |                                                                                           |
 | provisioning | Identifies businesses that are in the process of being activated for the Centrapay service.                                                 |                                                                                           |
 | active       | Identifies businesses that have been activated for Centrapay services successfully (i.e. they have successfully created a Payment Request). |                                                                                           |
 | deactivated  | Identifies businesses that have been de-registered for the Centrapay service.                                                               | duplicate, data-quality-issues, no-response, change-of-ownership, centrapay-discontinued. |
-| on-hold      | Identifies businesses that have been placed on hold.                                                                                        | seasonal-business-closure, incompatible-terminal.                                         |
+| on-hold      | Identifies businesses that have been placed on hold.                                                                                        | seasonal-business-closure.                                                                |
```

Comment:
@Ray2126 Let's remove this from the reasons table below too

Reply:
Done :)